packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -96,18 +96,20 @@ class SlidePanel extends React.Component {
       { fill },
       customProps.className,
     ]);
-
+    // Usage of tabIndex="0" to allow users to focus on potentially scrollable content
+    /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
     const panelDiv = (
-      <div className={cx(['panel'])} tabIndex="-1" aria-hidden={!isOpen ? 'true' : 'false'} ref={this.setPanelNode}>
```

Comment:
Did we add `-1` to ensure this div was never focusable?

Reply:
The update to axe-core for `Ensure scrollable region is focusable` seems to indicate that if an element supports scrolling via overflow styling, it should be focusable to allow users to interact with it to adjust the scrolling via keyboard. 

This is a new concept to me, however, I did find some accessibility articles that recommend this approach: https://developer.paciellogroup.com/blog/2016/02/short-note-on-improving-usability-of-scrollable-regions/

This is my first go at resolving the new axe-core errors, thinking on this some more, it may be better to apply this `tabIndex` to terra-scroll and use that as children inside of the slide-panel.